akka-stream-testkit/src/main/scala/akka/stream/testkit/StreamTestKit.scala

```diff
@@ -544,13 +566,35 @@ object TestSubscriber {
      * Fluent DSL
      *
      * Assert that no message is received for the specified time.
+     * NOTE! Uses old version of expectNoMsg with dilated timeout value.
      */
+    @deprecated(message = "Use expectNoMessage instead", since = "2.5.4")
     def expectNoMsg(remaining: FiniteDuration): Self = {
       probe.expectNoMsg(remaining)
       self
     }
 
     /**
+     * Fluent DSL
+     *
+     * Same as `expectNoMessage(testKitSettings.SingleExpectDefaultTimeout)`.
+     */
+    def expectNoMessage(): Self = {
```

Comment:
while we are at this we should only provide the one with `remaining` parameter, since almost always you don't want to wait the full default (or in scope) remaining timeout